.github/workflows/push_tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Run all tests
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  HF_HOME: /mnt/cache
+  OMP_NUM_THREADS: 8
+  MKL_NUM_THREADS: 8
+  PYTEST_TIMEOUT: 180
+  RUN_SLOW: yes
+
+jobs:
+  run_tests_single_gpu:
+    name: Diffusers tests
+    strategy:
+      fail-fast: false
+      matrix:
+        machine_type: [ single-gpu ]
+    runs-on: [ self-hosted, docker-gpu, '${{ matrix.machine_type }}' ]
+    container:
+      image: nvcr.io/nvidia/pytorch:22.07-py3
```

Comment:
Do you know which PyTorch version this is? Couldn't really find it here: https://catalog.ngc.nvidia.com/orgs/nvidia/containers/pytorch

Reply:
`PyTorch Version 1.13.0a0+08820cb`
Looks like all nvidia images are built from sources, so there are no stable versions. Although I can downgrade to `pytorch:22.05-py3` with `PyTorch Version 1.12.0a0+8a1a93a`

Reply:
I think Yeah PyTorch 1.13 is nightly-build or from source so I think PyTorch 1.12 would be the better choice here. In Transformers we automatically update the docker, but I think this is a bit too much here for now. Maybe let's better just do it manually in the beginning.

Reply:
In `transformers`, it's always the latest stable release PyTorch, unlesee it breaks something (and we pin the previous one for a while).